lib/gooddata/lcm/actions/collect_ldm_objects.rb

```diff
@@ -28,7 +28,7 @@ def call(params)
           synchronize = params.synchronize.pmap do |info|
             from = info.from
             from_project = development_client.projects(from) || fail("Invalid 'from' project specified - '#{from}'")
-            objects = (from_project.attributes.to_a + from_project.labels.to_a).map(&:uri)
+            objects = (from_project.attributes.to_a + from_project.labels.to_a + from_project.datasets.to_a + from_project.facts.to_a).map(&:uri)
```

Comment:
We should check that datasets and facts get transferred in the AppStore integration test (/spec/integration/shared_examples_for_synchronization_bricks.rb).